### ISSUE TITLE
Don't steal focus when showing reminder

### ISSUE THREAD

**Issue by coezbek** [ben-spiller/OutlookBetterReminders#9]
The current code steals focus from the active window when the reminder is shown. This isn't very nice when the user is typing and might lead to inadvertently closing the window (Pressing Enter while typing will Dismiss).

This also happens when the form is reactivated.

Is there a way to change this?

ChatGPT suggests the following change for the reactivation:

```
private void timerTick(object sender, EventArgs e)
{
    updateStartTime();

    // Close the form 2 hours after the meeting ends
    if (DateTime.Now > meeting.EndTime + new TimeSpan(2, 0, 0))
        Close();
    else if (DateTime.Now > reactivateTime)
    {
        // Update the next reactivation time
        reactivateTime = DateTime.Now + reactivateTimeSpan;

        // Restore the form if minimized, without forcing it to take focus
        if (WindowState == FormWindowState.Minimized)
        {
            logger.Debug("Restoring reminder window after timeout");

            // Restore the window without activating it
            ShowWindow(this.Handle, SW_RESTORE); // SW_RESTORE = 9, needs PInvoke setup
        }
    }
}

// You'll need to use Platform Invocation (PInvoke) to call ShowWindow.
[DllImport("user32.dll")]
private static extern bool ShowWindow(IntPtr hWnd, int nCmdShow);
```

**Comment by igitur** [comment]
I believe stealing the focus is one of the main goals of this plugin, since Outlook's standard popups appear in the background if Outlook isn't focussed. I specifically search around for a tool that steals the focus so that I don't miss meeting reminders anymore. Yesterday was quite embarrassing with I arrived 15min late in a meeting with some important people.

**Comment by ben-spiller** [comment]
Yes that's true, coming on top was a key part of why I made a new plugin since they're easily missed otherwise. I guess it could be made configurable for those who have a different preference - or we could look at other modern Windows notification mechanisms to attract attention. 

I'm not actively developing this project now that my comany doesn't use Outlook anymore, so it's up to others to pick up and torch and make/releases changes if there's something you'd like that it doesn't already do.

**Comment by igitur** [comment]
@ben-spiller I might take you up on that offer and start logging some PRs.
